gradle-nonnull-plugin/sample/.gitignore

```diff
@@ -7,3 +7,4 @@
 /build
 /captures
 .externalNativeBuild
+package-info.java
```

Comment:
That's how it was done before because we didn't know if it works through generated folder. It was in the sources. Should be removed now.